state/snapshotsManager.go

```diff
@@ -215,10 +211,7 @@ func (sm *snapshotsManager) prepareSnapshot(rootHash []byte, epoch uint32, trieS
 		return nil, true
 	}
 
-	defer func() {
-		err := trieStorageManager.PutInEpoch([]byte(lastSnapshot), rootHash, epoch)
-		handleLoggingWhenError("could not set lastSnapshot", err, "rootHash", rootHash)
-	}()
+	go sm.lastSnapshotMarker.AddMarker(trieStorageManager, epoch, rootHash)
```

Comment:
AddMarker will check internally for `WaitForStorageEpochChange` , should we avoid that in ImportDB processing mode?

Reply:
Indeed, there is no need for this if we run in importDB mode.